pkg/clients/aws/client.go

```diff
@@ -84,18 +84,33 @@ func (c *client) getAccountNumber(ctx context.Context) (string, error) {
 }
 
 var (
-	productSKUField         = "ProductSKU"
-	rancherProductSKU       = "0b87d4fa-d1fe-41d8-830b-67d4ec381549"
-	maxResults        int32 = 1
+	productSKUField       = "ProductSKU"
+	rancherProductSKUAmea = "0b87d4fa-d1fe-41d8-830b-67d4ec381549"
+	//TODO: This string is just a slight alteration and is not valid. Update with a valid value
```

Comment:
Is this a todo before merge, or a todo later?

Reply:
Before merge.

Reply:
But we still want to review it now so that the second we get this code we can merge (looking like we may not have the code until ~5 days before final RC and QA will still need to validate this).